docs/open-source-project/porting/system.md

```diff
@@ -100,6 +100,14 @@ To use most of the `resourced` functionalities, you must enable the following `c
 >
 > To use the `resourced` freezer feature, you must install the freezer plugin by enabling `CONFIG_FREEZER`.
 
+Resourced supports several swapping scenarios to provide more RAM to applications running in the foreground. To support them, enable the following options in the kernel configuration:
+
+  - `CONFIG_MEMCG_SWAP`: control swapping applications' memory, `CONFIG_SWAP` will be enabled automatically
+  - `CONFIG_ZRAM`: compressed block device
```

Comment:
Please rephrase to:

Compresses the block device.

Reply:
That is not what CONFIG_ZRAM means. CONFIG_ZRAM creates block devices (/dev/zram0, zram1 etc.) which compresses the received data and stores them in RAM.